service/dap/server.go

```
@@ -1000,6 +1007,9 @@ func (s *Server) onSetBreakpointsRequest(request *dap.SetBreakpointsRequest) {
 			response.Body.Breakpoints[i].Id = got.ID
 			response.Body.Breakpoints[i].Line = got.Line
 			response.Body.Breakpoints[i].Source = dap.Source{Name: request.Arguments.Source.Name, Path: clientPath}
+			if len(want.LogMessage) > 0 {
+				s.logPoints[got.ID] = want.LogMessage
```

Do we need to check if the log message is an acceptable format or not? The spec just says it is interpolated but we may need rules and documentation e.g. do we want to allow function calls, etc.?

And, I think it's better to respond with an error if the logpoint parsing fails, instead of reporting errors while handling the logpoints.